Mathlib/LinearAlgebra/Basis.lean

```diff
@@ -1247,8 +1247,9 @@ theorem coord_unitsSMul (e : Basis ι R₂ M) (w : ι → R₂ˣ) (i : ι) :
     trans ((unitsSMul e w).coord i) ((w j)⁻¹ • (unitsSMul e w) j)
     · congr
       simp [Basis.unitsSMul, ← mul_smul]
-    simp only [Basis.coord_apply, LinearMap.smul_apply, Basis.repr_self, Units.smul_def,
-      map_smul, Finsupp.single_apply]
+    simp only [Basis.coord_apply, LinearMap.smul_apply, Basis.repr_self, Units.smul_def]
+    simp only [map_smulₛₗ, RingHom.id_apply, repr_self, Finsupp.smul_single, smul_eq_mul,
+      mul_one, ne_eq, Finsupp.single_apply]
```

Comment:
This can also be reverted.
```suggestion
    simp only [Basis.coord_apply, LinearMap.smul_apply, Basis.repr_self, Units.smul_def,
      map_smul, Finsupp.single_apply]
```